src/User/UserContainer.jsx

```diff
@@ -10,35 +10,72 @@ const UserContainer = () => {
   const [user, setUser] = useState(null);
   const [isLoading, setIsLoading] = useState(true);
   const [error, setError] = useState(false);
+  const [isCurrentUser, setIsCurrentUser] = useState(false);
+  const [managerName, setManagerName] = useState("");
   const { id } = useParams();
   const [cookies] = useCookies([cookieName]);
 
+  const checkIfCurrentUser = (fetchedId, currentUserId) => {
+    if (Number(fetchedId) === currentUserId) {
+      setIsCurrentUser(true);
+    }
+  };
+
+  const getManagerName = async managerId => {
+    if (managerId) {
+      try {
+        const manager = await getUserById(
+          managerId,
+          cookies.sessionToken.token
+        );
+        setManagerName(`${manager.firstName} ${manager.lastName}`);
+      } catch (error) {
+        setError(true);
+      }
+    } else {
+      setManagerName("None Assigned");
```

Comment:
This should probably be a constant that can be read in other places, or as a generic label that can be overwritten in the point of display